### DISCUSSION TITLE
ChatGPT-Next-Web填写render部署后的api报错

### DISCUSSION THREAD

**Discussion by nowhereman1999** [linweiyuan/go-chatgpt-api#238] [Q&A]
如下图所示，ChatGPT-Next-Web填写render部署后的api报错：
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/53631206/f40b4b32-6292-4e12-b742-9a93794e46ba)
填写的接口地址是是http://go-chatgpt-api-nze3.onrender.com:8080/imitate，api是获取的access token,render服务器log输出如下：
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/53631206/e02287a7-6628-4193-a4ef-9c965902f866)
想知道是否因为接口地址填写错误，之前没用过render部署项目，不太清楚部署之后的api地址是多少，希望大佬们解惑！

**Comment by linweiyuan** [comment]
imitate + accessToken 不支持查钱

**Reply by nowhereman1999** [reply]
哦哦，那如果ChatGPT-Next-Web这个项目想使用这个代理，接口地址应该填什么呢

**Reply by linweiyuan** [reply]
不好意思忘了回，填到 /imitate，和自己的 access token

**Comment by nowhereman1999** [comment]
> http://go-chatgpt-api-nze3.onrender.com:8080/imitate

哦哦没事，但是我现在接口填的就是[http://go-chatgpt-api-nze3.onrender.com:8080/imitate](http://go-chatgpt-api-nze3.onrender.com:8080/imitate%EF%BC%8Capi%E6%98%AF%E8%8E%B7%E5%8F%96%E7%9A%84access) ，然后api key填的也就是access token但是不行

**Reply by linweiyuan** [reply]
访问不了这个链接

如果改成 https://go-chatgpt-api.linweiyuan.com/imitate，会怎样

**Comment by nowhereman1999** [comment]
> [https://go-chatgpt-api.linweiyuan.com/imitate](https://go-chatgpt-api.linweiyuan.com/imitate%EF%BC%8C%E4%BC%9A%E6%80%8E%E6%A0%B7)

![image](https://github.com/linweiyuan/go-chatgpt-api/assets/53631206/bac85dc6-b59a-403b-814a-13135e0bd16a)
改了之后可以，但是这样就不是用的我自己render搭建的接口吧

**Reply by linweiyuan** [reply]
这样就是用了我的线上服务，我是服务器部署的

你发的链接我打不开，不知如何测试

我那条 api 可以直接用

**Comment by nowhereman1999** [comment]
> > [https://go-chatgpt-api.linweiyuan.com/imitate](https://go-chatgpt-api.linweiyuan.com/imitate%EF%BC%8C%E4%BC%9A%E6%80%8E%E6%A0%B7)
> 
> ![image](https://user-images.githubusercontent.com/53631206/258978871-bac85dc6-b59a-403b-814a-13135e0bd16a.png) 改了之后可以，但是这样就不是用的我自己render搭建的接口吧

![image](https://github.com/linweiyuan/go-chatgpt-api/assets/53631206/b1fc76f5-a59d-4136-8886-9962c8243730)
刚刚把接口地址换成


> > [https://go-chatgpt-api.linweiyuan.com/imitate](https://go-chatgpt-api.linweiyuan.com/imitate%EF%BC%8C%E4%BC%9A%E6%80%8E%E6%A0%B7)
> 
> ![image](https://user-images.githubusercontent.com/53631206/258978871-bac85dc6-b59a-403b-814a-13135e0bd16a.png) 改了之后可以，但是这样就不是用的我自己render搭建的接口吧

刚刚把接口地址换成https://go-chatgpt-api-nze3.onrender.com/imitate之后也可以了：
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/53631206/654145a1-f915-4933-be56-3c59b8936c55)
这次应该是走的我render的接口了，之前应该就是多打了端口号，另外https改成了http所以不行，当时主要参考的这个仓库的readme：
![image](https://github.com/linweiyuan/go-chatgpt-api/assets/53631206/8156a496-ade7-42b3-9350-45f0a3ae8596)

**Reply by linweiyuan** [reply]
ok
